content/faq/_index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "FAQ"
+date: 2019-07-24
+draft: false
+disableToc: false
+---
+
+## Packit-as-a-Service
+
+### Can I use the packit service as soon as I install it into my repository?
+
+Thanks for your interest in Packit Service!  In order to start using the
+service you need to be whitelisted, which is an action to be done by us.  Once
+we put you on the whitelist, we'll get in touch with you.  We are now
+on-boarding Fedora contributors (with a Fedora Account System account).
+
+### Can I use packit service for any GitHub repository?
+
+Since packit service builds your PRs in Fedora COPR build service, by using
+Packit-as-a-service, your software needs to comply with [COPR
+rules](https://docs.pagure.org/copr.copr/user_documentation.html#what-i-can-build-in-copr).
+If any of these points are violated, we'll remove the builds and may put you on
+a blacklist so you won't be able to use the service again.
+
+### How can I contact you?
+
+If you encounter a problem while using Packit-as-a-service, please open an [upstream issue](https://github.com/packit-service/packit-service/issues/new). In case of any other questions, feel free to contact us on our mailing list: **user-cont-team@redhat.com**.
```

Comment:
```suggestion
If you encounter a problem while using Packit-as-a-service, please open an [upstream issue](https://github.com/packit-service/packit-service/issues/new). In case of any other questions, feel free to contact us: **user-cont-team@redhat.com**.
```

Reply:
that fact it's a ML is our implementation detail